test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -45,7 +44,14 @@ import (
 	"sigs.k8s.io/cluster-api/util/conditions"
 )
 
-var hookResponsesConfigMapName = "test-extension-hookresponses"
+// The Cluster API test extension uses a ConfigMap named cluster-name + suffix to determine answers to the lifecycle hook calls;
+// this tests uses this config map to test hooks blocking first, and then hook passing.
+
```

Comment:
```suggestion
// The Cluster API test extension uses a ConfigMap named cluster-name + suffix to determine answers to the lifecycle hook calls.
// This test uses this ConfigMap to test hooks blocking first, and then hooks passing.
```
(+ drop empty line, somehow wasn't able to do this via the suggestion)